Zed fails to start language server "oxc"

Hi, I just tried the `zed-oxc` extension but it doesn't work on my laptop.

Zed.log contains the following error:

```
2024-09-25T10:00:29.751856+02:00 [ERROR] failed to start language server "oxc": installed package 'oxlint' did not contain expected path '"./node_modules/@oxclint/darwin-arm64/oxc_language_server"'
```

The issue seems to be a wrong binary path reference, the parent folder should be `@oxlint` instead of  `@oxclint`.

This is very strange because the path seems right in the code:
https://github.com/suxiaoshao/zed-oxc/blob/d0d0d0ee5216c442590d12a6e90591b711eb20cb/src/oxc.rs#L12

I see there are no release tags for this extension, is it possible you have to republish it?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "zed-oxc"
-version = "0.1.0"
+version = "0.1.1"
 edition = "2021"
 license = "MIT"
 publish = false

--- a/extension.toml
+++ b/extension.toml
@@ -1,10 +1,10 @@
 authors = ["suxiaoshao <https://github.com/suxiaoshao>"]
 description = "Oxc support for Zed"
 id = "oxc"
-name = "oxc"
+name = "Oxc"
 repository = "https://github.com/suxiaoshao/zed-oxc"
 schema_version = 1
-version = "0.1.0"
+version = "0.1.1"
 
 [language_servers.oxc]
 code_actions_kind = ["", "quickfix"]

--- a/src/oxc.rs
+++ b/src/oxc.rs
@@ -12,7 +12,7 @@ use zed_extension_api::{
 const SERVER_PATH: &str = "node_modules/oxlint/bin/oxc_language_server";
 const PACKAGE_NAME: &str = "oxlint";
 
-const OXC_CONFIG_PATHS: &[&str] = &[".oxlintrc.json"];
+const OXC_CONFIG_PATHS: &[&str] = &["oxlintrc.json"];
 
 struct OxcExtension;
 
@@ -25,7 +25,7 @@ impl OxcExtension {
         let (platform, arch) = zed_extension_api::current_platform();
 
         Ok(format!(
-            "@oxclint/{platform}-{arch}/oxc_language_server",
+            "@oxlint/{platform}-{arch}/oxc_language_server",
             platform = match platform {
                 zed::Os::Mac => "darwin",
                 zed::Os::Linux => "linux",
@@ -59,11 +59,12 @@ impl OxcExtension {
 
         if server_package_exists {
             let worktree_root_path = worktree.root_path();
-
-            return Ok(Path::new(worktree_root_path.as_str())
+            let path = Path::new(worktree_root_path.as_str())
                 .join(SERVER_PATH)
                 .to_string_lossy()
-                .to_string());
+                .to_string();
+            zed::make_file_executable(&path);
+            return Ok(path);
         }
 
         // fallback to extension owned biome
@@ -157,7 +158,7 @@ impl zed_extension_api::Extension for OxcExtension {
                 args.push(config_path.clone());
             } else if require_config_file {
                 return Err(
-                    ".oxlintrc.json is not found but require_config_file is true".to_string(),
+                    "oxlintrc.json is not found but require_config_file is true".to_string()
                 );
             }
         }

